web/assets/frontend/images/logo.svg

```diff
@@ -8,10 +8,10 @@
+                    <path d="M22.9417249,32.0792541 L20.5202797,29.6578089 L27.2135198,22.9417249 L20.5202797,16.225641 L22.9417249,13.8041958 L32.0792541,22.9417249 L22.9417249,32.0792541 Z M13.8041958,22.9417249 L4.66666667,13.8041958 L13.8041958,4.66666667 L16.225641,7.08811189 L9.50955711,13.8041958 L16.225641,20.5202797 L13.8041958,22.9417249 Z" id="Shape" fill="#0000A0"></path>
+                    <path d="M36.3636364,18.1818182 L32.9545455,18.1818182 C32.9545455,10.0454545 26.3409091,3.40909091 18.1818182,3.40909091 L18.1818182,0 C28.2045455,0 36.3636364,8.15909091 36.3636364,18.1818182 Z M18.1818182,36.3636364 C8.15909091,36.3636364 0,28.2045455 0,18.1818182 L3.40909091,18.1818182 C3.40909091,26.3181818 10.0227273,32.9545455 18.1818182,32.9545455 L18.1818182,36.3636364 Z" id="Shape" fill="#00BD46"></path>
```

Comment:
Test image change in assets? Can you explain more please.

Reply:
these commits are for testing purposes only, They will be dropped.